### DISCUSSION TITLE
How to specify ssl certificate in piccolo.conf file for postgres connection?

### DISCUSSION THREAD

**Discussion by sumitsharansatsangi** [piccolo-orm/piccolo#800] [Q&A] [answered]
I have to use ssl certificate ( for testing , it is now self-signed) for database connection,  I didn't find anything on the documentation , how can I specify?

**Comment by dantownsend** [comment, marked as answer]
Have a look here:

https://magicstack.github.io/asyncpg/current/api/index.html#connection

`PostgresEngine` takes a config argument which is passed straight through to `asyncpg`.

Have a look at the `ssl` argument, or alternatively the `dsn` argument.
